src/idd/idd.base.js

```diff
@@ -1152,15 +1177,15 @@ var _initializeInteractiveDataDisplay = function () { // determines settings dep
             var _tooltip; // <div> element which displays the tooltip
             var _updateTooltip;
 
-            var foreachDependentPlot = function (plot, f) {
-                var myChildren = plot.children;
-                var n = myChildren.length;
-                for (var i = 0; i < n; i++) {
-                    var child = myChildren[i];
-                    foreachDependentPlot(child, f);
-                }
-                f(plot);
-            };
+            //var foreachDependentPlot = function (plot, f) {
```

Comment:
@ElenaPochernina Why did you leave the commented line? I suggest removing them if they are not needed.